Editor/BootstrapperConfig.cs

```diff
@@ -16,6 +16,11 @@ static class BootstrapperConfig
 
         public const string GitHubRepositoryURL = @"ssh://git@github.com:StansAssets/OneSignal-Unity-SDK.git";
         public const string BootstrapperFolderPath = @"Assets/OneSignalBootstrap/Editor";
```

Comment:
Shouldn't this clean up the whole `Assets/OneSignalBootstrap` folder, not just the `Editor`?

Reply:
agreed